src/components/Pager/index.js

```diff
@@ -19,7 +19,7 @@ class Pager extends React.Component {
 
   render() {
     const currentPage = this.props.page;
-    const totalPages = Math.floor(this.props.itemCount / 100) + 1;
+    const totalPages = Math.floor(this.props.itemCount / this.props.pageSize) + 1;
     const hasPrevious = currentPage > 1;
```

Comment:
There are some issues with pagination. Pagination is also already done by backend. They return a 'count' and an previous and next link. We could simply use those values to determine what is needed on the frontend. Probably best to have an offline discussion about this.